Announcements schema

AndrewBoland opened the pull request:
Fixes # .
#105 

### PR Description
Implements schema for announcements

### What needs to be reviewed?
server/db/model/Announcement.js

### Context?
I wasn't sure which queries to implement so I added a few, feedback as to which to take out, modify or add would be great.

connorkrupp commented on server/db/model/Announcement.js:
```diff
@@ -2,3 +2,72 @@
+    });
+};
+
+// Allow us to query by token
```
Update this comment you silly goose

connorkrupp commented on server/db/model/Announcement.js:
```diff
@@ -2,3 +2,72 @@
+};
+
+// Initialize the model with the schema, and export it
+var model = mongoose.model('Announcements', schema);
```
Make 'Announcements' singular

connorkrupp commented on server/db/model/Announcement.js:
```diff
@@ -2,3 +2,72 @@
+        isSent: true
+    });
+};
+
```
Let's make a query for ones that are both approved and sent and call it `isPublic` or something better if you think of a better way to express it

AndrewBoland commented:
I updated the byIsPublic query to query for time before now. I also fixed all the comments. Let me know if there is anything else I should do or if I should rename anything.
